spec/controllers/posts/create_spec.rb

```diff
@@ -128,12 +131,15 @@
       expect(assigns(:post).settings.size).to eq(0)
       expect(assigns(:post).content_warnings.size).to eq(0)
       expect(assigns(:post).labels.size).to eq(0)
+      expect(assigns(:post).fonts.size).to eq(0)
       expect(assigns(:settings).map(&:id_for_select)).to match_array(settings.map(&:id) + ['_other'])
       expect(assigns(:content_warnings).map(&:id_for_select)).to match_array(warnings.map(&:id) + ['_other'])
       expect(assigns(:labels).map(&:id_for_select)).to match_array(labels.map(&:id) + ['_other'])
+      expect(assigns(:fonts).map(&:id)).to match_array(fonts.map(&:id))
       expect(Setting.count).to eq(2)
       expect(ContentWarning.count).to eq(2)
       expect(Label.count).to eq(2)
+      expect(Font.count).to eq(2)
```

Comment:
I think you actually want to check PostFont.count is 0 here (to make sure it failed, rather than making sure the factory worked)